packages/invoices/src/InboundShipment/DetailView/Toolbar.tsx

```diff
@@ -73,7 +73,7 @@ export const Toolbar: FC = () => {
             />
           </Box>
         </Grid>
-        <DropdownMenu disabled={isDisabled} label={t('label.select')}>
+        <DropdownMenu label={t('label.select')}>
```

Comment:
I've always had a problem with 'Select' here.. what do you think about changing the label to 'Actions' instead?
The word select doesn't really tell me what is going to happen when I click it 🤷

Reply:
Yeah good idea: 3863acf2